src/ec/packages/ec-component-site-header-harmonised/demo/data--group3.js

```diff
@@ -0,0 +1,12 @@
+import specs from '@ecl/ec-specs-site-header-harmonised/demo/data--group3';
```

Comment:
i remember we talked about an image that you were setting in the story to be moved here, wasn't needed, that's the reason why i don't see it here?